cve_bin_tool/checkers/ncurses.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for Ncurses
+
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-72/product_id-38464/GNU-Ncurses.html
+
+RSS feed: http://www.cvedetails.com/vulnerability-feed.php?vendor_id=72&product_id=38464&version_id=0&orderby=3&cvssscoremin=0
+
+"""
+from ..util import regex_find
+
+
+def guess_contains_ncurses(lines):
+    """Tries to determine if a file includes Ncurses
+    """
+    for line in lines:
+        if "_nc_first_name" in line:
+            return 1
+    return 0
+
+
+def get_version(lines, filename):
+    """returns version information for Ncurses as found in a given file.
+    The version info is returned as a tuple:
+        [modulename, is_or_contains, version]
+
+    modulename will be ncurses if Ncurses is found (and blank otherwise)
+    is_or_contains idicates if the file is a copy of Ncurses or contains one
+    version gives the actual version number
+
+    VPkg: gnu, ncurses
+    """
+    regex = [r"ncurses ([0-9]+\.[0-9]+)"]
```

Comment:
Should we extend this to expect a date at the end of the string, the way it appears in your compiled versions?  (We don't have to use the date, but it's there every time it's an addtional way to reduce false positives).

Reply:
We could do that, but there are some versions on rpmfind's page which don't have a date. Its probably safer to keep it like this.